docs/source/specific-guides/network_nat/sip_tcp.rst

```diff
@@ -80,10 +80,15 @@ It could be the case that the initial request is sent with TCP, but the
 subsequent ones are with UDP. In this case, check the URI in the *Route*
 or *Record-Route* or *Contact* header of the 18x or 2xx response that is
 sent by the remote party. Chances are this header lacks the
-``“;transport=tcp"`` parameter in the URI; in this case, you can either
-configure the other end to use TCP, or configure your proxy to
-*record-route* (i.e. to force itself to be within the request path of
-the call).
+``“;transport=tcp"`` parameter in the URI; in this case, you can take one
+of the following measures:
+
+ - Configure the other end to use TCP,
+ - Configure your proxy to *record-route* (i.e. to force itself to be
+   within the request path of the call).
+ - Configure the transport of the accounts to have explicit control
+   (:cpp:any:`AccountSipConfig::transportId`/:cpp:any:`Account::setTransport()`
+   or :cpp:any:`pjsua_acc_config::transport_id`/:cpp:any:`pjsua_acc_set_transport()`).
```

Comment:
Actually the account's transport binding is for different purposes (e.g: selecting IP version or network interface) and cannot be used for specifying transport type for sending a SIP request. The transport type is specified in the SIP request message itself (e.g: via target URI or proxy/route) as described in the docs. So if you send a request using transport type that is different to the binding (e.g: account is bound to UDP while request specifies a TCP target), the request will fail to send.